cli.js

```diff
@@ -19,10 +19,12 @@ if (argv.help || argv.h) {
 Usage:
     geojson-to-svg [-p mercator]
 Options:
-    --projection  -p  Which projection to use. Default: mercator
-                      Refer to npmjs.com/projections for a list.
+	--projection  -p	Which projection to use. Default: mercator
```

Comment:
The indentation is still different.

Reply:
Ok, I'll fixit

Reply:
Thanks! 💛

Reply:
Maybe VSCode is messing with that, I'll try another editor